cmd/main.go

```diff
@@ -75,5 +75,15 @@ func startServer(config *config.Config, airbrakeNotifier *gobrake.Notifier, logg
 	if server.TLSConfig, err = config.ExtractServiceTLSConfig(logger); err != nil {
 		return err
 	}
-	return server.ListenAndServeTLS(config.TLS.ServerCert, config.TLS.ServerKey)
+	if err := server.ListenAndServeTLS(config.TLS.ServerCert, config.TLS.ServerKey); err != nil {
```

Comment:
This will always return a non-nil error.

https://cs.opensource.google/go/go/+/refs/tags/go1.23.3:src/net/http/server.go;l=3540-3541